data/json/items/ammo.json

```
@@ -811,7 +814,7 @@
   {
     "type": "AMMO",
     "id": "rebreather_filter",
-    "category": "spare_parts",
+    "category": "tools_entry",
```

Missed one of the filter items.
```suggestion
    "category": "spare_parts",
```